admin/tools.php

```diff
@@ -8,8 +8,9 @@
 yourls_html_menu();
 ?>
 
-	<div class="sub_wrap">
-	
+	<div class="sub_wrap">	
+	<main role="main">
```

Comment:
Ok for this one
I wanted to keep the div that has a position in YOURLS css, because of HTML5 support in old browser. But it's better if you merge the two.
But you can add support with a js&css shiv : https://github.com/afarkas/html5shiv